cmd/up/exec_windows.go

```diff
@@ -32,9 +29,32 @@ func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*
 
 	c.Dir = he.workdir
 
+	// https://docs.studygolang.com/pkg/syscall/?GOOS=windows#SysProcAttr
 	c.SysProcAttr = &syscall.SysProcAttr{
+		// SysProcAttr for windows are different from linux,
+		// use CreationFlags to group the processes
 		CreationFlags: syscall.CREATE_NEW_PROCESS_GROUP,
 	}
 
 	return c, nil
 }
+
+func expandExecutableInCurrentDirectory(args0, dir string) (string, error) {
```

Comment:
NIT: I'd go with `binaryName` instead of `args0`, but both of them are readable for me
```suggestion
func expandExecutableInCurrentDirectory(binaryName, dir string) (string, error) {
```